fsd_config/form_jsons/dpif_r3/dpif-declarations.json

```diff
@@ -0,0 +1,158 @@
+{
+    "startPage": "/declarations",
+    "pages": [
+        {
+            "path": "/summary",
+            "title": "Check your answers",
+            "components": [],
+            "next": [],
+            "section": "HgUinB",
+            "controller": "./pages/summary.js"
+        },
+        {
+            "path": "/declarations",
+            "title": "Declarations",
+            "components": [
+                {
+                    "type": "Html",
+                    "content": "<p class=\"govuk-body govuk-!-font-weight-bold\">In this section, we'll ask you to:</p>\n<ul class=\"govuk-list govuk-list--bullet\">\n<li>confirm you\u00a0have signed the Local Digital Declaration and agree to follow the 5 principles</li>\n<li>confirm your section 151 officer consents to the funds being carried over and spent in the next financial year (March 2025 to April 2026) and beyond, if deemed necessary in project budget planning</li>\n<li>confirm you agree to let all outputs from this work be published under open licence with a view to any organisation accessing, using or adopting them freely</li>\n<li>confirm the information you have provided is accurate</li>\n</ul>",
```

Comment:
Is this unicode character intentional? `\u00a0`?

Reply:
No idea, this is FAB's export..

@nuwan-samarasinghe @NarenderRajuB @wjrm500 any idea?

Reply:
I don't want to delay deployment, as the deadline is on Wednesday. If the unicode char is a problem, Forms team can look into it separately.